packages/node_modules/@webex/react-component-activity-item-base/src/index.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-
+import {FormattedMessage} from 'react-intl';
```

Comment:
Is it public component? is it same as web-client?

Reply:
**Yes, It is same as web-client, we are using FormattedMessage React Component from 'react-intl' library to implement internationalization, we are doing same thing for formatting delete message as well.**